files/image_config/hostcfgd/hostcfgd

```diff
@@ -356,7 +361,15 @@ class HostConfigDaemon:
             syslog.syslog(syslog.LOG_WARNING, "Enable state of feature '{}' is None".format(feature_name))
             return
 
-        self.update_feature_state(feature_name, state, feature_table)
+        # Compare the current value of 'state' field with cached value, if they are not equal, the function
+        # 'update_feature_state(...)' will be invoked to enable/disable the corresponding container service.
+        # At the same time, the cached value will be updated with current value of 'state' field. The reason
+        # to cache the previous value of 'state' field and do the comparison at here is that if we issue the 
+        # 'config' command to update the value of 'auto_restart' field in 'FEATURE' table, the function 
+        # 'update_feature_state(...)' will also be invoked and this behavior is not expected.
```

Comment:
This comment is a bit overly-verbose. It could be simplified into a single sentence

Reply:
Great suggestion! Will summarize with a single sentence.